examples/nextjs-blog/pages/index.js

```diff
@@ -1,9 +1,174 @@
-function Home() {
+import React from 'react'
+// import Carousel from 'nuka-carousel'
+import PropTypes from 'prop-types'
+import {
+  Grid,
+  Card,
+  CardActions,
+  CardMedia,
+  CardContent,
+} from '@material-ui/core'
+import { flamelinkApp as app } from '../utils/flamelink'
+import { Link } from '../routes'
+import { getImageAlt, getDateString } from '../utils/post/post.util'
+import { DEFAULT_POST_IMAGE_URL } from '../constants/constants'
+import { containerWide } from '../components/styled'
+
+const Home = function(props) {
+  function renderPostCards(posts) {
+    return posts.map(post => {
+      const {
+        title,
+        slug,
+        author: { displayName },
+        excerpt,
+        _fl_meta_: { docId },
+        image,
+        date,
+      } = post
+
+      const { url } = (image && image[0]) || {
+        url: DEFAULT_POST_IMAGE_URL,
+      }
+
+      return (
+        <Grid key={docId} item xs={12} md={6} lg={4} xl={3}>
```

Comment:
I would suggest refactoring this into a separate reusable component that we can use on the listing page as well. Then we can just pass it an array of posts that it will render.

Reply:
Okay, will do when I update the code post SDK fix. 👍

Reply:
SDK should be fixed now. I've bumped the version here